### ISSUE TITLE
Sanshikan tests

### ISSUE THREAD

**Issue by NiclasHedam** [Siteimprove/alfa#83]
After #73, Sanshikan rules are not considered testable and should have unit-tests.

**Comment by NiclasHedam** [comment]
This issue will be on hold for now. 
I have written tests to Sanshikan R1-R18 in #82 and #84, but since Sanshikan R19-R28 are not documented upstream yet, I cannot continue.
